.vale/styles/RedHat/TermsErrors.yml

```diff
@@ -303,3 +303,27 @@ swap:
   wish: want
   wish|would like: want
   zero out: zero
+  byte code: bytecode
+  client: customer
+  computer farm|computer ranch: server cluster
+  details pane: details view
+  domain mode: managed domain
+  enum: data enumeration
+  gather: collect
+  live broker: master broker
+  live only: live-only
+  load-balance|load-balancing: load balance
+  multi site|multi-site: multisite
+  package: packet
```

Comment:
Rules like https://redhat-documentation.github.io/supplementary-style-guide/#packet are context-specific. In essence, it says, "when you're talking about discrete units of data, say _packet_ instead of _package_." Unfortunately, Vale can't determine what the writer's intention or full context was (yet!) and creates a false positive when it encounters a valid instance of "package", such as when you're talking about a "software package".

Are there criteria that can help us identify other style rules in this PR that follow a similar pattern:
- The replacement word is a common word that likely has good uses.
- The SSG rule is context-specific, not general.

For example:

- Line 307: `client: customer` This rule is valid when the content describes _people_ using products. But it generates false positives when the content describes client-server patterns in software, such as "Registering an OAuth client". See https://redhat-documentation.github.io/supplementary-style-guide/#collect

- Line 312: `gather: collect` This rule is valid for content about the RHV log collector. But it generates false positives in a more general context, such as "so you can gather data about more than one feature". See https://redhat-documentation.github.io/supplementary-style-guide/#collect

Should we simply remove rules like these, or is there a way we might get them to work?

Reply:
Thanks Rolfe! Yeah I figured we'd end up removing a good chunk of these. 

I don't think we need to spend much time on it tbh - if the rules are too general and add noise when caught in the wrong context then we just remove it. We can note the terms that don't fit the Vale rule set for future reference I guess.

Reply:
@rolfedh I've updated the PR taking your suggestions on board to remove any terms that can be ambiguous in meaning depending on the context. 

Removed the following swap terms:

```json
[
    "collect: gather",
    "engine: inference engine",
    "package: packet",
    "product: Product"
]
```

Reply:
Let me know if you see any other problematic terms in the PR.